packages/svelte-ux/src/lib/utils/number.ts

```diff
@@ -1,81 +1,93 @@
-import { format as d3Format } from 'd3-format';
+import { getFormatNumberOptions } from '$lib/components/setup';
+
+export type FormatNumberStyle =
+  | 'decimal' // from Intl.NumberFormat options.style NumberFormatOptions
+  | 'currency' // from Intl.NumberFormat options.style NumberFormatOptions
+  | 'percent' // from Intl.NumberFormat options.style NumberFormatOptions
+  | 'unit' // from Intl.NumberFormat options.style NumberFormatOptions
+  | 'none'
+  | 'integer'
+  | 'percentRound'
+  | 'metric';
+
+export type FormatNumberOptions = Intl.NumberFormatOptions & {
+  style?: FormatNumberStyle;
+  locales?: string | undefined;
+  fractionDigits?: number;
+};
 
 // See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat
 export function formatNumber(
   number: number | null | undefined,
-  options: Intl.NumberFormatOptions & { fractionDigits?: number } = {}
+  options: FormatNumberOptions & {
+    suffix?: string;
```

Comment:
Why are `suffix` and `suffixExtraIfMany` not part of `FormatNumberOptions`?

Reply:
Heuuuuu, was late yesterday. Thx for looking 💪
😅